### PR TITLE
AAP-35119 updated overview to tested topologies

### PR DESCRIPTION
**Pull request by emurtoug**
[AAP-35119](https://issues.redhat.com/browse/AAP-35119)

Updated introductory paragraph to [Chapter 1. Overview of tested deployment models](https://docs.redhat.com/en/documentation/red_hat_ansible_automation_platform/2.5/html/tested_deployment_models/overview-tested-deployment-models)

Files modified:
- assembly-overview-tested-deployment-models.adoc

### CODE REVIEW

File: downstream/assemblies/topologies/assembly-overview-tested-deployment-models.adoc
Comment (jself-sudoku) [inline downstream/assemblies/topologies/assembly-overview-tested-deployment-models.adoc:5]: Suggestion based on RH supplementary guide regarding using "Red Hat suggests": 

```suggestion
Red Hat tests {PlatformNameShort} {PlatformVers} with a defined set of topologies to give you opinionated deployment options. Based on results from these tests, always deploy all components of {PlatformNameShort} so that all features and capabilities are available for use without the need to take further action.
```

File: downstream/assemblies/topologies/assembly-overview-tested-deployment-models.adoc
Comment (jself-sudoku) [inline downstream/assemblies/topologies/assembly-overview-tested-deployment-models.adoc:7]: Same as above and based on RH supplementary guide: 

```suggestion
Install {PlatformNameShort} {PlatformVers} based on a defined set of infrastructure topologies or reference architectures. Enterprise organizations should use one of the enterprise topologies for production deployments to ensure the highest level of uptime, performance, and continued scalability. Organizations or deployments that are resource constrained can use a "growth" topology.
```

File: downstream/assemblies/topologies/assembly-overview-tested-deployment-models.adoc
Comment (jself-sudoku) [inline downstream/assemblies/topologies/assembly-overview-tested-deployment-models.adoc:9]: Suggestion to adhere to rules about "Red Hat recommends": 

```suggestion
It is possible to install the {PlatformNameShort} on different infrastructure topologies and with different environment configurations. Red Hat does not fully test topologies outside of published reference architectures. Use a tested topology for all new deployments.  
```

File: downstream/assemblies/topologies/assembly-overview-tested-deployment-models.adoc
Comment (g-murray) [inline downstream/assemblies/topologies/assembly-overview-tested-deployment-models.adoc:7]: Hey I'd recommend avoiding "should" here, maybe "can" or "must" instead?

IBM SG: "Whenever possible, rewrite in a more direct way. Do not use should to refer to an action that must be performed. "

**Review by g-murray — APPROVED** [review]
LGTM

File: downstream/assemblies/topologies/assembly-overview-tested-deployment-models.adoc
Comment (jself-sudoku) [reply downstream/assemblies/topologies/assembly-overview-tested-deployment-models.adoc]: @g-murray  - Yes, I considered adding that to the edits, but wording, "Whenever possible..." gave me pause. Tried the different options and felt like "should" was tolerable in this instance. Leaving it up to @emurtoug if she wants to make further changes after this PR is merged.

**Pull request merged**
